packages/analytics-js-integrations/src/integrations/AdobeAnalytics/heartbeatHandle.js

```
@@ -138,7 +139,8 @@
   let adSessionCount = adBreakCounts[session_id || 'deafult'];
   adSessionCount = adSessionCount
     ? adBreakCounts[session_id || 'default'] + 1
-    : (adBreakCounts[session_id || 'default'] = 1);
+    : // eslint-disable-next-line sonarjs/no-nested-assignment
+      (adBreakCounts[session_id || 'default'] = 1);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)